ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/Headers.hs

```diff
@@ -0,0 +1,77 @@
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -Wno-missing-export-lists #-}
```

Comment:
We don't usually enable this option. I don't know if it is enabled anywhere else.